packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx

```diff
@@ -30,6 +30,7 @@ const propTypes = {
 const defaultProps = {
   altText: 'Client Logo',
   isCard: false,
+  url: undefined,
```

Comment:
this isn't needed is it? should be undefined anyway

Reply:
yup by default props will be undefined. [updated](https://github.com/cerner/terra-consumer/pull/60/commits/a79dc69228e7473e8638480a670e14cd48dfb274)